psutil/_pswindows.py

```diff
@@ -244,15 +244,21 @@ def swap_memory():
     mem = cext.virtual_mem()
 
     total_phys = mem[0]
-    free_phys = mem[1]
     total_system = mem[2]
-    free_system = mem[3]
 
     # system memory (commit total/limit) is the sum of physical and swap
     # thus physical memory values need to be substracted to get swap values
     total = total_system - total_phys
-    free = min(total, free_system - free_phys)
-    used = total - free
+    # commit total is incremented immediately (decrementing free_system)
+    # while the corresponding free physical value is not decremented until
+    # pages are accessed, so we can't use free system memory for swap.
+    # instead, we calculate page file usage based on performance counter
+    if (total > 0):
+        percentswap = cext.getpercentswap()
+        used = int(0.01 * percentswap * total)
+    else:
+        used = 0
+    free = total - used
     percent = usage_percent(used, total, round_=1)
```

Comment:
Shouldn't you use `percentswap` directly instead of passing through `usage_percent()`?

Reply:
But math is fun!

To be fair, that code was already there, I was just fixing how to calculate `used` and `free`...